LondonBritishLibrary/orient/BLorient5268.xml

```diff
@@ -0,0 +1,400 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BLorient5268" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Psalter</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 5268</idno>
+                        <altIdentifier><idno>Strelcyn 25</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                    <material key="paper"><locus target="#I"/></material>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">1+163</measure>
+                                    <measure unit="leaf" type="blank" corresp="#I">1</measure>
+                                </extent>
+                            </supportDesc>    
+                        </objectDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <locus target="#Iv"/>
+                                    <desc type="Unclear">Palimpsest on the verso of the front flyleaf.</desc>
```

Comment:
Is this really a palimpsest? The leaf is made of paper. MAybe it is an erased note?

Reply:
You are right, this information given by Strelcyn is somehow contradictory and confusing. He says, that there is one unnumbered flyleaf, which is unnumbered and that the "paper flyleaves (!) were supplied, when the manuscript was repaired in the B.M.". If I think about it once again, I come to the conclusion, that there are one (parchment) flyleaf (with a palimpsest) + (at least two) paper flyleaves. I will update the record accordingly. I do not assume, that Strelcyn counted an erased portion of a text as a palimpsest.